.github/workflows/build-tests.yaml

```diff
@@ -0,0 +1,18 @@
+name: Run Tonomy-SDK Tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: 10.x
```

Comment:
we want node version 16.14.1, same as what everyone is using. perhaps we can use a docker container with this node version?

Reply:
Okay, I'll update the version.